app/models/lti_configuration.rb

```diff
@@ -3,6 +3,8 @@
 class LtiConfiguration < ApplicationRecord
   belongs_to :organization
 
+  validates :lms_type, presence: true
```

Comment:
Won't this throw exceptions for all LMS configurations that don't currently have a lms_type? ActiveRecord will run validations any time a record is saved. We may need a backfill here first.

Reply:
Within the migration we specified a default LMS type (`:other`) that automatically backfilled any that was previously nil, as it's a required column on postgres now.

Reply:
Oh awesome, I didn't know we had a default value.